app/Http/Controllers/AppRegisterController.php

```diff
@@ -159,9 +162,32 @@ public function onboarding(Request $request)
 
         sleep(10);
```

Comment:
Whats the purpose of this sleep?

Reply:
After the User is created, the UserObserver generates the public/private key, and associated models. Normally we don't need this, but since we are immediately generating an authToken and calling verify_credentials in the app.